src/ast/stmt/SwitchStmt.php

```diff
@@ -1,93 +0,0 @@
-<?php
```

Comment:
switch will be replaced by `match`, right?

Reply:
Yes!